### PR TITLE
Team awesome

### PR DESCRIPTION
**Pull request by Zurina**

### CODE REVIEW

File: teams/1/LandingPage.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+      <div class="members-nav">
+        <a href="#">Login</a>
+        <a href="#">Sign up</a>
+      </div>
+
+    </nav>
+  </header>
+  <main>
+    <header>
+      <h1>Grow your users. <br />
```
Comment (Zurina) [inline teams/1/LandingPage.html:46]: I guess we should remove the <br />'s

File: teams/1/LandingPage.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+      <!-- not sure to use form  --> <!--I wouldnt use a form, as it is just a single input, not really a form-->
+      <span>
+        <input type="text" name="email" required placeholder="Enter your email" />
+        <label>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus, reprehenderit.</label>
+      </span>
+
+      <input type="submit" value="Get Started" />
+
+      <div>
+        <p>Users over time</p>
```
Comment (Zurina) [inline teams/1/LandingPage.html:60]: This could be changed to a span

File: teams/1/LandingPage.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+    <section id="data">
+      <h2>Unleash your full power of data</h2>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur autem omnis alias dolorem sapiente
+        fugiat dolore neque libero quod laudantium.</p>
+      <div>
+        <!-- duplicate 3 times -->
+        <div>
+          <strong>40+</strong>
+          <p>Integrations</p>
```
Comment (Zurina) [inline teams/1/LandingPage.html:110]: Span as well

File: teams/1/LandingPage.html [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+    <section id="pricing">
+      <span>Pricing</span>
+      <h2>Plans that fit your scale</h2>
+      <p>Simple, transparent pricing that grows with you. Try and plan free for 30 days.</p>
+      <div>
+        <!-- copy of this -->
+        <div>
+          <div>
+            <h3>Basic plan</h3>
+            <p>Our most popular plan for small features</p>
```
Comment (Zurina) [inline teams/1/LandingPage.html:147]: Could be either a span or a paragraph I guess - I'd say span though

File: teams/1/LandingPage.html [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+    <hr>
+
+    <article id="faq">
+      <h2>Frequently asked questions</h2>
+      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Temporibus, nisi.</p>
+      <!-- collapsibles: needed 6 times -->
+      <div class="faq-card">
+
+        <div class="faq-card-question-1">
+          <p><strong>Is there a free trial available?</strong></p>
```
Comment (Zurina) [inline teams/1/LandingPage.html:183]: Span

File: teams/1/LandingPage.html [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+      </div>
+
+        <div class="faq-more-questions">
+          <div class="users-pic-faq">
+            <img src="" alt="">
+            <img src="" alt="">
+            <img src="" alt="">
+          </div>
+          <p><strong>Still have questions?</strong></p>
```
Comment (Zurina) [inline teams/1/LandingPage.html:197]: Span

File: teams/1/Contacts.html [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">San Francisco</h2>
+                        <p>100 Market Street <br> San Francisco, CA 94105 USA</p>
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">Sweden</h2>
+                        <p>Drottniggatan 100 <br> 111 60 Stockholm SE</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:138]: Span

File: teams/1/Contacts.html [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">London</h2>
+                        <p>100 Oxford Street <br> London W1D 1LL UK</p>
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">San Francisco</h2>
+                        <p>100 Market Street <br> San Francisco, CA 94105 USA</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:133]: Spam

File: teams/1/Contacts.html [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">Byron Bay</h2>
+                        <p>100 Jonson Street <br> Byron Bay NSW 2481 AU</p>
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">London</h2>
+                        <p>100 Oxford Street <br> London W1D 1LL UK</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:128]: Span

File: teams/1/Contacts.html [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">Sydney</h2>
+                        <p>100 George Street <br> Sydney NSW 2000 AU</p>
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">Byron Bay</h2>
+                        <p>100 Jonson Street <br> Byron Bay NSW 2481 AU</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:123]: Span

File: teams/1/Contacts.html [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+
+                <ul>
+                    <li>
+                        <h2 class="office-city">Melbourne</h2>
+                        <p>100 Flinders Street <br> Melbourne VIC 3000 AU</p>
+                    </li>
+
+                    <li>
+                        <h2 class="office-city">Sydney</h2>
+                        <p>100 George Street <br> Sydney NSW 2000 AU</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:118]: Span

File: teams/1/Contacts.html [hunk 12]
```diff
@@ -0,0 +1,328 @@
...
+                <h4>Find us at these locations</h4>
+            </header>
+
+            <!--Offices List-->
+            <article class="location-offices">
+
+                <ul>
+                    <li>
+                        <h2 class="office-city">Melbourne</h2>
+                        <p>100 Flinders Street <br> Melbourne VIC 3000 AU</p>
```
Comment (Zurina) [inline teams/1/Contacts.html:113]: Span

File: teams/1/Contacts.html [hunk 13]
```diff
@@ -0,0 +1,328 @@
...
+
+        </section>
+
+        <!--Locations Section-->
+        <section class="our-locations">
+
+            <header class="location-title">
+                <h2>Our Locations</h2>
+                <h3>Visit our stores</h3>
+                <h4>Find us at these locations</h4>
```
Comment (Zurina) [inline teams/1/Contacts.html:104]: Maybe span instead of header?

File: teams/1/Contacts.html [hunk 14]
```diff
@@ -0,0 +1,328 @@
...
+        </footer>
+
+        </section>
+
+        <!--Locations Section-->
+        <section class="our-locations">
+
+            <header class="location-title">
+                <h2>Our Locations</h2>
+                <h3>Visit our stores</h3>
```
Comment (Zurina) [inline teams/1/Contacts.html:103]: Maybe span instead of header?

File: teams/1/Contacts.html [hunk 15]
```diff
@@ -0,0 +1,328 @@
...
+
+            <article class="map-info sales">
+                <h2>Sales</h2>
+                <h3>Questions or queries? Get in touch</h3>
+                <a href="mailto:#">sales@untitledui.com</a>
+            </article>
+
+            <article class="map-info phone">
+                <h2>Phone</h2>
+                <h3>Mon - Fri from 8am to 5pm.</h3>
```
Comment (Zurina) [inline teams/1/Contacts.html:90]: Could be a span as well

File: teams/1/Contacts.html [hunk 16]
```diff
@@ -0,0 +1,328 @@
...
+
+            <article class="map-info support">
+                <h2>Support</h2>
+                <h3>Our friendly team is here to help.</h3>
+                <a href="mailto:#">support@untitledui.com</a>
+            </article>
+
+            <article class="map-info sales">
+                <h2>Sales</h2>
+                <h3>Questions or queries? Get in touch</h3>
```
Comment (Zurina) [inline teams/1/Contacts.html:84]: Possible span?

File: teams/1/Contacts.html [hunk 17]
```diff
@@ -0,0 +1,328 @@
...
+                </div>
+                
+            </figure>
+
+                <!--Map Footer Contact Info-->
+        <footer class="map-footer">
+
+            <article class="map-info support">
+                <h2>Support</h2>
+                <h3>Our friendly team is here to help.</h3>
```
Comment (Zurina) [inline teams/1/Contacts.html:78]: Possible span?

File: teams/1/Contacts.html [hunk 18]
```diff
@@ -0,0 +1,328 @@
...
+                        <div class="location-circle Byron-Bay">Byron-Bay</div>
+                        <div class="location-circle London">London</div>
+                        <div class="location-circle San-Francisco">San Francisco</div>
+                        <div class="location-circle Sweden">Sweden</div>
+
+                <!--Box Figure of Clicked Location-->
+                <div class="location-box">
+                    <span class="box-flag"></span>
+                    <strong>Melbourne, AUS</strong>
+                    <p>100 Smith Street <br> Collingwood VIC 3086 AU </p>
```
Comment (Zurina) [inline teams/1/Contacts.html:67]: Possible span?

File: teams/1/Dashboard.html [hunk 19]
```diff
@@ -0,0 +1,201 @@
+<!doctype html>
```
Comment (Scientist-Ansh) [inline teams/1/Dashboard.html:1]: code looks good but please fix the indentation

**Review by Scientist-Ansh — APPROVED** [review]

**Review by tonyblu331 — COMMENTED** [review]
Review changes done

**Review by juliusberthold — APPROVED** [review]

**Review by nikabutikashvili — APPROVED** [review]

**Review by tonyblu331 — APPROVED** [review]

**Pull request merged**